src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,129 @@
+@import "../utils/functions.scss";
```

Comment:
Suggestion: it might be better to connect all files to main.scss 🤓

Reply:
@snuber-w Thanks, done.

_I accidentally resolved the conversation_